.github/workflows/test.yml

```diff
@@ -43,14 +43,30 @@ jobs:
       - name: Test with unittest
         shell: bash -el {0}
         run: |
+          # Unit tests
+          rm -rf ~/.cache/mlagility
+          python test/database.py
+
+          # Examples
+          rm -rf ~/.cache/mlagility
```

Comment:
fyi @danielholanda I added these `rm` statements to make sure every test was starting from a clean cache.

Reply:
I like the intention, but I don't know if we are actually doing much here. 
If you look at `test/cli.py `you will see that the cache is manually set to `cache_dir = "cache-dir"` (not `~/.cache/mlagility`).
Additionally, `test/cli.py` is also already ensuring that the cache dir is empty before starting:

```
if dirpath.is_dir():
    shutil.rmtree(dirpath)
```

Reply:
It definitely applies to the database test and the Example tests, since those will all use the default cache directory unless otherwise specified (and none of them do).

It will also catch the case where someone deleted the lines you're referencing from the E2E tests.

Consider it a meta-test?

Reply:
I can remove it from the E2E tests if you want, but I am resolute it should stay with the Examples and database tests.

Reply:
No need to remove it from E2E tests. If we are doing this I agree we should be consistent.